packages/app/src/features/savings/components/general-stats-bar/GeneralStatsBar.stories.ts

```diff
@@ -38,7 +38,7 @@ export const HighValues: Story = {
     generalStatsResult: {
       data: {
         tvl: NormalizedUnitNumber(10_000_000_000),
-        getLiquidity: () => NormalizedUnitNumber(Number.POSITIVE_INFINITY),
+        getLiquidityCap: () => NormalizedUnitNumber(Number.POSITIVE_INFINITY),
```

Comment:
```suggestion
        getLiquidityCap: () => undefined
```
Otherwise stories are unrealistic